.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Connect to KOTSADM",
+            "type": "go",
+            "request": "attach",
+            "mode": "remote",
+            "port": 40000,
+            "host": "127.0.0.1"
+        },
+    ]
+}
```

Comment:
I know this file is gitignored, but IMHO I don't know why you wouldn't include it for posterity